pagespeed/apache/mod_instaweb.cc

```diff
@@ -1789,6 +1790,8 @@ static const command_rec mod_pagespeed_filter_cmds[] = {
   APACHE_CONFIG_OPTION(kModPagespeedBlockingRewriteRefererUrls,
                        "wildcard_spec for referer urls which trigger blocking "
                        "rewrites"),
+  APACHE_CONFIG_OPTION(kModPagespeedFollowFlushes,
+      "Attempt to mirror incoming flushes for html streams in the output"),
```

Comment:
... Though the description is likely to be confusing, since mod_pagespeed doesn't generally use ProxyFetch, and I am pretty sure InstawebContext does pass through all the Apache flushes unconditionally.

(Also this doesn't follow the style guide --- can't have one line with arguments indented and another at +4).